src/test/java/it/pagopa/pn/delivery/pnclient/mandate/PnMandateClientImplTest.java

```diff
@@ -13,11 +13,12 @@
 import org.springframework.http.RequestEntity;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.DefaultUriBuilderFactory;
 
 import java.util.Collections;
 import java.util.List;
 
-class PnMandateClientImplTestIT {
+class PnMandateClientImplTest {
```

Comment:
È stato scritto come testo unitario, mockando anche il RestTemplate, quindi ho eliminato il suffisso IT.